storybook/html/alert-banner-success.html

```diff
@@ -0,0 +1,6 @@
+<div class="ayr-alert-banner ayr-alert-banner--success"
```

Comment:
can we not use the macros instead of putting raw html in these stories? Otherwise these can easily get out of sync with our actual components used in the webapp?